src/gromacs/nblib/topology.h

```diff
@@ -69,6 +69,19 @@ std::vector<gmx::ExclusionBlock> toGmxExclusionBlock(const std::vector<std::tupl
 // Add offset to all indices in inBlock
 std::vector<gmx::ExclusionBlock> offsetGmxBlock(std::vector<gmx::ExclusionBlock> inBlock, int offset);
 
+// extract all bonds of type B from a vector of molecules. The second argument tuple element
```

Comment:
These comments are somehow less clear than the code itself. I think the only way to document these functions is with an explanation of this sort, but more expanded like from what you said in the slack. Also, the docs should include the examples you type out in the tests of these functions. I no longer think that this code is necessarily going to be too complicated to maintain. However, we will do our future selves a favor to have a quite long winded explanation of how these functions work.